### DISCUSSION TITLE
Question / Support - Moving to a Domain from IP:PORT

### DISCUSSION THREAD

**Discussion by EpsilonCommunity** [cockpit-project/cockpit#16556] [Q&A] [answered]
Hi,

Looked on the wiki and the documentation with no clear path to enable change from local IP to domain.

Any help would be greatly appreciated or linking to an overlooked page.

**Comment by martinpitt** [comment, marked as answer]
What do you mean with this exactly? IP/Domain is a function of your DNS, not cockpit. Can you please clarify your question? Thanks!

**Reply by martinpitt** [reply]
No further reply, marking this as an answer to clean up the unanswered list.
